robot_smach_states/examples/Lethal_route_planning.py

```diff
@@ -0,0 +1,81 @@
+# ROS
+import PyKDL as kdl
+import rospy
+import math
+from std_msgs.msg import Bool
+from geometry_msgs.msg import Twist
+from nav_msgs.msg import OccupancyGrid
+
+# TU/e Robotics
+from robot_skills.get_robot import get_robot
+
+
+class Lethal_Zone:
+    def __init__(self, robot):
+        self.robot = robot
+        self.front_bumper_sub = rospy.Subscriber("hero/base_f_bumper_sensor", Bool, self.front_callback)
+        self.front_bumper_active = False
+        self.back_bumper_sub = rospy.Subscriber("hero/base_b_bumper_sensor", Bool, self.back_callback)
+        self.back_bumper_active = False
+        self._cmd_vel = rospy.Publisher("hero/base/references", Twist)
+        self.global_costmap_sub = rospy.Subscriber("/hero/global_planner/global_costmap/costmap", OccupancyGrid, self.global_costmap_callback)
+        self.costmap_info = None
+        self.costmap_data = None
+
+    def front_callback(self, msg):
+        rospy.loginfo(msg)
+        self.front_bumper_active = msg.data
+
+    def back_callback(self, msg):
+        rospy.loginfo(msg)
+        self.back_bumper_active = msg.data
+
+    def global_costmap_callback(self, msg):
+        rospy.loginfo(msg)
```

Comment:
Only log for development. This will really slow down performance as the global costmap is pretty big

Reply:
Even for development this will not realy be helpful. It will just flood your screen with numbers